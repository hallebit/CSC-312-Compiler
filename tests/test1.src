(if (<= 5 6) 
    (+ 1 (- 8 6)) 
    (/ 5 2))