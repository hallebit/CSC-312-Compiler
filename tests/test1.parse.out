(if (5 <= 6) then ((1 + 8) - 6) else (5 / 2))